README.md

```diff
@@ -18,7 +18,7 @@ The Pipeline Model Definition Plugin provides a config-like syntax for defining
 - To start developing plugin: 
 - Clone the plugin repository
 
-`cd pipeline-model-definition && mvn install && mvn hpi:run`
+`cd pipeline-model-definition-plugin && mvn install && mvn hpi:run`
```

Comment:
This doesn't work as-is because you can't run `hpi:run` from the root of a multi-module repo:
```suggestion
`cd pipeline-model-definition-plugin && mvn install && mvn -f pipeline-model-definition/pom.xml hpi:run`
```

Reply:
```suggestion
`cd pipeline-model-definition && mvn install && mvn hpi:run`
```